pkg/catalog/routes.go

```diff
@@ -130,6 +127,11 @@ func (mc *MeshCatalog) GetWeightedClusterForService(svc service.MeshService) (se
 	return getDefaultWeightedClusterForService(svc), nil
 }
 
+// hostnamesTostr returns a comma separated string of hostnames from the list
+func hostnamesTostr(hostnames []string) string {
+	return strings.Join(hostnames, ",")
+}
+
 // GetHostnamesForService returns the hostnames for a service
 func (mc *MeshCatalog) GetHostnamesForService(meshService service.MeshService) (string, error) {
```

Comment:
should we rename this `GetHostnamesandPortsforService`?
or maybe we could document this in a comment. was unclear to me that the ports were also being extracted at first.

Reply:
Comment is more appropriate since port can be part of the hostname.